docs/en/reference/namingstrategy.rst

```diff
@@ -21,6 +21,12 @@ You can specify a different strategy by calling ``Doctrine\ORM\Configuration#set
     <?php
     $namingStrategy = new MyNamingStrategy();
     $configuration->setNamingStrategy($namingStrategy);
+    
+ 
+
+Warning
+
+The naming strategy is always overridden by annotations on an entity.
```

Comment:
To render a nice warning block we should format this like:

```rst
.. warning

    The naming strategy is always overridden by annotations on an entity.
```

---

I'd also suggest moving this block to the intro (right before of the title`Configuring a naming strategy`) to make things a bit more explicit.
Text-wise I think we could a bit more verbose since I noticed that we do mention this on the [intro "when the column or table name is not given"](https://github.com/doctrine/orm/pull/8041/files#diff-88b1a8bc0ff7545ed3ba546bf87a03d4R7).

Perhaps removing the "when the column or table name is not given" and saying something like "Naming strategies are completely ignored when table or column names are provided in the entity mapping", what do you think?